### ISSUE TITLE
About OceanPlus's training.

### ISSUE THREAD

**Issue by buoucatlabuladuodog** [researchmm/TracKit#72]
Hello there, this is really an impressive work. I want to ask a few questions about training. Whether the training data set is processed in the same way as SiamMask? During training, is there any sequence or mixed use of the two data sets of YouTube-VOS and COCO? Is the setting of the loss function similar to that of SiamMask, or only considers the loss of the predicted mask？ Do you have any other suggestions for training? Looking forward to your reply.

**Comment by JudasDie** [comment]
> Hello there, this is really an impressive work. I want to ask a few questions about training. Whether the training data set is processed in the same way as SiamMask? During training, is there any sequence or mixed use of the two data sets of YouTube-VOS and COCO? Is the setting of the loss function similar to that of SiamMask, or only considers the loss of the predicted mask？ Do you have any other suggestions for training? Looking forward to your reply.

Hi, thanks for your interest. 1) the data is processed as SiamMask 2) in each epoch, we use 20w pairs for COCO and 40w pairs for YTBVOS 3) the loss function is classical BCE 4) training code will be released once published. Hope this information can help you.

**Comment by buoucatlabuladuodog** [comment]
Thank you very much for your reply！
